src/constants.js

```diff
@@ -24,12 +24,18 @@ export const SAMPLED_MASK = 0x1;
 // DEBUG_MASK is the bit mask indicationg that a span has been marked for debug.
 export const DEBUG_MASK = 0x2;
 
+// DEFERRED_SAMPLING_MASK is the bit mask indicating that a span has been marked for deferred sampling.
```

Comment:
the first sentence is a bit of a tautology. 

DEFERRED_SAMPLING_MASK is the bit mask indicating that the upstream service that generated the span did not make a definitive sampling decision, but deferred it to be made by some downstream service.

No need to mention mobile, imo, especially since the claim that we cannot have a feedback loop with them is not exactly true.